nss-cohort-36/keahua-arboretum-net-surfers
# Fix/kikakapu

# Description

Changed the spellings of animals to their true Hawaiian naming.

# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout fix/kikakapu`
4. Annex a biome then go to the `Release animal` menu.
5. Make sure you see Kikakapu as `Kīkākapu`
6. Make sure you see Ulae as `'Ulae`
7. As always, make sure nothing BREAKS 💯 

# Checklist:
